vnext/src/Libraries/Network/RCTNetworking.windesktop.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./RCTNetworkingWinShared');
```

Comment:
Do we have any convention for naming files like this right now?

Reply:
no convention I can find.  This seems fine to me.

---
In reply to: [372656571](https://github.com/microsoft/react-native-windows/pull/3983#discussion_r372656571) [](ancestors = 372656571)